### PR TITLE
Google-Button

### PR DESCRIPTION
**Pull request by AlaaTaima**
Relates #5

### CODE REVIEW

File: src/components/GoogleButton/index.js [hunk 1]
```diff
@@ -10,32 +10,30 @@ import './style.css';
...
+        {
+          email: userInfo.email,
+          name: userInfo.name,
+          goal: '',
+          userID: socialAuthUser.user.uid,
+          createdByGoogle: true,
+        },
+        { merge: true }
+      );
+      await localStorage.setItem('userId', socialAuthUser.user.uid);
```
Comment (yosefanajjar) [inline src/components/GoogleButton/index.js]: We don't need await

File: src/components/GoogleButton/index.js [hunk 2]
```diff
@@ -10,32 +10,30 @@ import './style.css';
...
+      this.setState({ error: null });
+      await push(ROUTES.HOME);
```
Comment (yosefanajjar) [inline src/components/GoogleButton/index.js]: we don't need await

File: src/components/TwitterButton/index.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ class SignInTwitter extends Component {
     ) : (
       <button type="submit" className="facebook-btn" onClick={this.handleClick}>
         <FacebookImg className="facebook-btn__img" />
-        <span className="facebook-btn__text">Facebook</span>
+        <span className="facebook-btn__text">Twitter</span>
```
Comment (Fatmasiam) [inline src/components/TwitterButton/index.js]: move the error msg from the button

Comment (Fatmasiam) [inline src/components/TwitterButton/index.js]: add twitter logo not facebook

File: src/containers/Firebase/firebase.js [hunk 4]
```diff
@@ -41,6 +41,10 @@ class Firebase {
   user = uid => this.db.doc(`users/${uid}`);
 
   users = () => this.db.collection('users');
+
+  journal = Jid => this.db.doc(`journals/${Jid}`);
```
Comment (Fatmasiam) [inline src/containers/Firebase/firebase.js]:
```suggestion
  journal = journalId => this.db.doc(`journals/${journalId}`);
```

**Pull request merged**